server/controllers/AuthController.js

```diff
@@ -87,7 +87,7 @@ const oauthRedirectFromGithubController = async (req, res) => {
     return loginSucceeded();
 };
 
-const signoutController = (req, res) => {
+const signoutController = async (req, res) => {
```

Comment:
What's this change for!

Reply:
An unnecessary change now :). I promisified the `destroy` in its own method to use elsewhere in the `AuthController` as well but there was no real benefit to it. Forgot to remove the async keyword here.